app/helpers/admin/tabbed_nav_helper.rb

```diff
@@ -44,7 +44,7 @@ def tab_navigation(tabs, *extra_classes)
 
   def tab_dropdown(label, menu_items)
     content_tag(:li, class: 'dropdown') do
-      content_tag(:a, class: 'dropdown-toggle', :'data-toggle' => 'dropdown', href: '#') do
+      content_tag(:a, class: 'dropdown-toggle', 'data-toggle': 'dropdown', href: '#') do
```

Comment:
Interesting, didn't know the hash-syntax supported this for symbols that need quotes.  TIL.